frontend/src/styles.css

```diff
@@ -3,57 +3,23 @@
 @tailwind components;
 @tailwind utilities;
 
-.btn-file {
-    position: relative;
-    overflow: hidden;
-}
-
-.btn-file input[type=file] {
-    position: absolute;
-    top: 0;
-    right: 0;
-    min-width: 100%;
-    min-height: 100%;
-    font-size: 100px;
-    text-align: right;
-    filter: alpha(opacity=0);
-    opacity: 0;
-    outline: none;
-    background: white;
-    cursor: inherit;
-    display: block;
+html{
+    font-size: 62.5%; 
```

Comment:
Is this 62.5% a best practice number? Where does it come from?

Reply:
We can remove it, wrong copy /paste from the starter. It was used time ago for responsive fonts